metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -230,28 +247,56 @@ func CreateStackdriverSink(uri *url.URL) (core.DataSink, error) {
 		return nil, err
 	}
 
+	requestQueue := make(chan *sd_api.CreateTimeSeriesRequest)
+
 	sink := &StackdriverSink{
 		project:           projectId,
 		zone:              zone,
 		stackdriverClient: stackdriverClient,
+		requestQueue:      requestQueue,
+		workers:           workers,
 	}
 
 	// Register sink metrics
 	prometheus.MustRegister(requestsSent)
 	prometheus.MustRegister(timeseriesSent)
 
+	// Launch Go routines responsible for sending requests
+	for i := 0; i < sink.workers; i++ {
+		go sink.requestSender(sink.requestQueue)
+	}
+
 	glog.Infof("Created Stackdriver sink")
 
+	glog.Infof("Workers: %v", sink.workers)
```

Comment:
Please combine this message with the previous one and explain what type of worked you mean.